### PR TITLE
Import existing crontab jobs

### PR DESCRIPTION
**Pull request by gabriellhrn**
I've added an option to import existing jobs from a crontab file (I thought it would be a good feature when I messed my crontab file and had to rewrite it from the cron.log 😞).

The screenshots below show how it should work.

screenshot from 2016-03-12 16-29-44
screenshot from 2016-03-12 16-31-55
screenshot from 2016-03-12 16-32-32
screenshot from 2016-03-12 16-33-15
screenshot from 2016-03-12 16-33-37
screenshot from 2016-03-12 16-34-32

### CODE REVIEW

**Comment by mvlbarcelos** [comment]
+1

**Comment by jamesrwhite** [comment]
Related to: https://github.com/jamesrwhite/minicron/issues/111

File: lib/minicron/cron.rb [hunk 1]
```diff
@@ -96,6 +97,75 @@ def build_crontab(host)
...
+      # Read all the file
+      crontab = conn.exec!("crontab -l")
+
+      # Parse the content
+      crontab_jobs = []
+      crontab.to_s.split("\n").each do |line|
+        line.strip
+
+        # We only want lines starting with a number or '*' (ignoring spaces)
+        next if line.empty? or line !~ /^\s*[0-9*]/
```
Comment (jamesrwhite) [inline lib/minicron/cron.rb]: What about "special" schedules? e.g. `@reboot`, `@hourly` etc

File: lib/minicron/hub/controllers/jobs.rb [hunk 2]
```diff
@@ -459,4 +459,87 @@ class Minicron::Hub::App
       erb :'jobs/schedules/delete', :layout => :'layouts/app'
     end
   end
+
+  get '/jobs/import/:host' do
```
Comment (jamesrwhite) [inline lib/minicron/hub/controllers/jobs.rb]: Could this be moved to the `hosts` controller and have the route as `/hosts/:id/import` instead? I think this more closely follows how the other routes work.

**Comment by jamesrwhite** [comment]
Aside from the two comments above this looks great, thank you!

I might make a few tweaks post merge to make it clearer to the user what this actually does and possibly have a step that shows you what jobs would be imported.

**Comment by gabriellhrn** [comment]
I forgot about the special strings. I've put them now.

It makes more sense in the hosts controller indeed. Moved it.

**Comment by jamesrwhite** [comment]
Thanks!

**Pull request merged**